server/athenian/api/openapi/openapi.yaml

```diff
@@ -1435,3 +1513,327 @@ components:
+          description: Commit message.
+          type: string
+        size_added:
+          description: Overall number of lines added.
+          type: integer
+        size_removed:
+          description: Overall number of lines removed.
+          type: integer
+        files_changed:
+          description: Number of files changed in this PR.
+          type: integer
+      required:
+      - author
+      - committer
+      - files_changed
+      - hash
+      - message
+      - repository
+      - size_added
+      - size_removed
+      type: object
+    CommitSignature:
+      description: Git commit signature.
+      example:
+        login: github.com/vmarkovtsev
+        name: Vadim Markovtsev
+        email: vadim@athenian.co
+        timestamp: 2020-02-25T14:27:21Z
+      properties:
+        login:
+          $ref: '#/components/schemas/DeveloperID'
+        name:
+          description: Git signature name.
+          type: string
+        email:
+          description: Git signature email.
+          type: string
+        timestamp:
+          description: When the corresponding action happened.
+          format: date-time
+          type: string
+      required:
+      - email
+      - name
+      - timestamp
+      type: object
+    CommitFilter:
+      description: Filter for committed code.
+      example:
+        account: 1
+        date_from: 2020-01-01
+        date_to: 2020-03-01
+        granularity: day
+        in:
+        - '{1}'
+        with_author:
+        - github.com/vmarkovtsev
+      properties:
+        account:
+          description: Session account ID.
+          type: integer
+        date_from:
+          description: Commits must be made later than or equal to this date.
+          format: date
+          type: string
+        date_to:
+          description: Commits must be made earlier than or equal to this date.
+          format: date
+          type: string
+        granularity:
+          $ref: '#/components/schemas/Granularity'
+        in:
+          $ref: '#/components/schemas/RepositorySet'
+        with_author:
+          $ref: '#/components/schemas/DeveloperSet'
+        with_committer:
+          $ref: '#/components/schemas/DeveloperSet'
+      required:
+      - account
+      - date_from
+      - date_to
+      - granularity
+      - in
+      type: object
+    MeasuredCodeBypassingPRs:
+      description: Statistics about code pushed outside of pull requests.
+      example:
+      - date: 2020-01-23
+        bypassed_commits: 1
+        bypassed_lines: 20
+        total_commits: 2
+        total_lines: 50
+      - date: 2020-01-24
+        bypassed_commits: 2
+        bypassed_lines: 30
+        total_commits: 3
+        total_lines: 60
+      items:
+        $ref: '#/components/schemas/CodeBypassingPRsMeasurement'
+      type: array
+    CodeBypassingPRsMeasurement:
+      description: Statistics about code pushed outside of pull requests in a
+        certain time interval.
```

Comment:
_ditto_
it is not needed to mention the time interval since it depends on the query building it